samples/csharp/MicrosoftEduGraphSamples/Workflows/AssignmentWorkflow.cs

```diff
@@ -123,5 +123,41 @@ public async Task AssignmentsEvolvableEnumsAsync()
                 Console.WriteLine($"AssignmentsEvolvableEnumsAsync: {ex.ToString()}");
             }
         }
-    }
+
+        /// <summary>
+        /// Workflow to get assignments from all the classes which are not archived
+        /// </summary>
+        /// <param name="isTeacher">True value accepts Teacher account and false for Student account</param> 
```

Comment:
revise the parameter name and the description

Reply:
updated the parameter name and description for appOnly